modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -34,7 +34,8 @@
 			'includeHeader': true,
 			'renderWhenEmpty': false,
 			'paging': false,
-			'pageSize': 25
+			'pageSize': 25,
+			'showEmptyPlaylistError': true
 		},
```

Comment:
@eitanavgil  - no need to define it here if also defined in mw.KBaseMediaList.js as playlistAPI inherits it.